components/ImprovedSearch/PaginationItem.tsx

```diff
@@ -0,0 +1,93 @@
+import { usePagination } from 'react-instantsearch-hooks-web'
+import { Button } from "./Button"
+import styled from 'styled-components'
+
+
+const StyledListItem = styled.li`
+  display: inline-block;
+`
+
+const PaginationLink = styled(Button)<{ isCurrent?: boolean; inverted?: boolean }>`
+  width: 44px;
+  height: 44px;
+  color: ${(props) => (props.inverted ? 'hsl(184, 31%, 74%)' : 'hsl(206, 32%, 21%);')}
```

Comment:
We should use design tokens from EDS for the colour variables here

Reply:
I tried using tokens.colors.interactive.primary__resting.hex but was throwing error  and i was not sure exactly which one should i use here .

Reply:
Should be doable, we use ESS tokens for colours other places in the code if you search. Exactly which one to use it's always a bit hard to figure out 😅 But this one `tokens.colors.interactive.primary__resting.hex` should work, I use it for the StepperItem 🤔